src/java/com/zimbra/qa/unittest/TestSSDBEphemeralStore.java

```diff
@@ -35,298 +39,279 @@
     private String SAMPLE_AUTH_TOKEN_VERSION = "8.7.0_GA_1659";
     private String ACCOUNT_ID = "47e456be-b00a-465e-a1db-4b53e64fa";
     private boolean SSDBStoreConfigured = false;
+    private List<Pair<EphemeralInput, EphemeralLocation>> toDelete = new ArrayList<Pair<EphemeralInput, EphemeralLocation>>();
+
     @Override
     public void setUp() throws Exception {
         String ssdbUrl = Provisioning.getInstance().getConfig().getEphemeralBackendURL();
         String toks[] = ssdbUrl.split(":");
         if(toks != null && toks.length == 3 && "ssdb".equalsIgnoreCase(toks[0])) {
-            SSDBStoreConfigured = true;
             EphemeralStore.setFactory(SSDBEphemeralStore.Factory.class);
-            store = SSDBEphemeralStore.getFactory().getStore();
-            SSDBEphemeralStore.getFactory().startup();
+            EphemeralStore.Factory factory = EphemeralStore.getFactory();
+            factory.test(ssdbUrl);
+            store = factory.getStore();
+            assertTrue(store instanceof SSDBEphemeralStore);
```

Comment:
Please add a useful string so that we can see at a glance why this test failed (if it does) instead of having to dig into it.